client/src/assets/scss/main.scss

```diff
@@ -12,3 +12,23 @@
   -moz-tab-size: 4;
   tab-size: 4;
 }
+
+.show {
+  display: flex;
+  margin: 100px auto 0;
+  width: 60%;
+  flex-direction: column;
+  border: 2px solid brown;
+  padding: 20px;
+  border-radius: 25px;
+  background: url(https://i.pinimg.com/originals/2a/66/b7/2a66b7151e30f47e3c4f430c25902a41.jpg);
+  text-align: center;
+}
+
+body {
```

Comment:
Best practice would be to define more global styles at the top - this `body` is applied throughout the application